client/web/compose/src/components/Admin/Module/UniqueValues.vue

```diff
@@ -0,0 +1,210 @@
+<template>
+  <div>
+    <div class="d-flex justify-content-end">
+      <b-button
+        size="lg"
+        variant="link"
+        class="d-flex align-items-center text-decoration-none p-0 mb-5"
+        @click="addNewConstraint"
+      >
+        {{ $t("addNewConstraint") }}
+        <font-awesome-icon
+          :icon="['fas', 'plus']"
+          size="sm"
+          class="ml-1"
+        />
+      </b-button>
+    </div>
+
+    <div
+      v-for="(rule, index) in rules"
+      :key="index"
+    >
+      <hr v-if="index">
+      <div>
+        <div class="d-flex justify-content-between align-items-center">
+          <h5>{{ $t("uniqueValueConstraint") }} #{{ index + 1 }}</h5>
+
+          <div class="px-4">
+            <c-input-confirm
+              variant="link"
+              size="sm"
```

Comment:
The constraints delete icon should be md and both should look like the field table ones